pkgs/io_file/constants.json

```diff
@@ -0,0 +1,52 @@
+{
+  "<sys/stat.h>": [
+    "S_IEXEC",
+    "S_IFBLK",
+    "S_IFCHR",
+    "S_IFDIR",
+    "S_IFIFO",
+    "S_IFLNK",
+    "S_IFMT",
+    "S_IFREG",
+    "S_IFSOCK",
+    "S_IFWHT",
+    "S_IREAD",
+    "S_IRGRP",
+    "S_IROTH",
+    "S_IRUSR",
+    "S_IRWXG",
+    "S_IRWXO",
+    "S_IRWXU",
+    "S_ISGID",
+    "S_ISTXT",
+    "S_ISUID",
+    "S_ISVTX",
+    "S_IWGRP",
+    "S_IWOTH",
+    "S_IWRITE",
+    "S_IWUSR",
+    "S_IXGRP",
+    "S_IXOTH",
+    "S_IXUSR",
+    "UF_APPEND",
+    "UF_HIDDEN"
+  ],
+  "<errno.h>": [
+    "EACCES",
+    "EEXIST",
+    "EINTR",
+    "ENOENT",
+    "EPERM",
+    "O_APPEND",
+    "O_CLOEXEC",
+    "O_CREAT",
+    "O_EXCL",
+    "O_RDONLY",
+    "O_TRUNC",
+    "O_WRONLY"
+  ],
+  "<fcntl.h>": [
+    "AT_FDCWD",
+    "AT_REMOVEDIR"
+  ]
+}
```

Comment:
nit: newline

Reply:
The json pretty printer doesn't like newlines for some reason.